File: Banking-Kata/src/test/java/banking/model/AccountTest.java
package banking.model;

import static org.assertj.core.api.Assertions.assertThat;

import org.junit.Test;

public class AccountTest {

  @Test
  public void canSpecifyAStartingBalance() throws Exception {
    Account account = new Account(Money.of("65.98"));

    assertThat(account.getBalance()).isEqualTo(Money.of("65.98"));
  }
  
  @Test
  public void eachAccountHasAUniqueId() throws Exception {
    Account account1 = new Account(Money.of("1.23"));
    Account account2 = new Account(Money.of("1.23"));
    
    assertThat(account1.getId()).isNotEqualTo(account2.getId());
  }

}

File: 07-Banking-Kata/csharp-dotnetcore2/BankingKataAPI.Test/Models/AccountTest.cs
using BankingKataAPI.Models;
using Xunit;

namespace BankingKataAPI.Test.Models
{
    public class AccountTest
    {
        [Fact]
        public void CanSpecifyAStartingBalance()
        {
            var account = new Account(new Money(65.98m));

            Assert.Equal(account.Balance, new Money(65.98m));
        }

        [Fact]
        public void ShouldGenerateIdForAccount()
        {
            var account1 = new Account(new Money());
            var account2 = new Account(new Money());

            Assert.NotEqual(account1.Id, account2.Id);
        }
    }
}
